sickrage/providers/GenericProvider.py

```diff
@@ -121,11 +123,28 @@ def download_result(self, result):
 
         return False
 
-    def find_propers(self, search_date=None):
-        results = self.cache.listPropers(search_date)
+    def find_propers(self, proper_candidates):
+        results = []
+
+        for proper_candidate in proper_candidates:
+            show_obj = Show.find(sickbeard.showList, int(proper_candidate[b'showid'])) if proper_candidate[b'showid'] else None
+
+            if show_obj:
+                episode_obj = show_obj.getEpisode(proper_candidate[b'season'], proper_candidate[b'episode'])
+
+                for term in self.proper_strings:
+                    search_strings = self._get_episode_search_strings(episode_obj, add_string=term)
 
-        return [Proper(x[b'name'], x[b'url'], datetime.fromtimestamp(x[b'time']), self.show, x[b'seeders'],
-                       x[b'leechers'], x[b'size'], x[b'pubdate'], x[b'hash']) for x in results]
+                    for item in self.search(search_strings[0], ep_obj=episode_obj):
```

Comment:
Im on mobile. Can you do a force proper search in develop with a torrent provider that doesnt support OR search? That code is the same from develop